### PR TITLE
Added info about FFZ not working with the new Twitch site

### PR DESCRIPTION
**Pull request by Gei0r**
Now that the twitch legacy site is no longer available, a lot of people will probably come and ask why FFZ doesn't work anymore.

I added a little explanation to the Readme file about the situation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,21 @@ Copyright (c) 2017 Dan Salvato LLC
 
 Licensed under the Apache License, Version 2.0. See LICENSE.
 
+FrankerFaceZ Doesn't Work After Twitch Site Update
```
Comment (pahakalle) [inline README.md]: Well it works but not all the features, maybe add the list from the update to this

Comment (pahakalle) [reply README.md]: Or from here https://trello.com/b/LGcYPFwi/frankerfacez-v4

Comment (Gei0r) [reply README.md]: I don't know what you mean by "the list from the update"

Comment (pahakalle) [reply README.md]: Oh i mean when you open the control center on twitch, it has a list of the features on the home page of it.

**Comment by pahakalle** [comment]
Seems like this PR is not needed anymore

**Comment by NBruderman** [comment]
I think this can be closed now

**Pull request closed without merging**